compiler/optimizer/Inliner.cpp

```diff
@@ -4519,6 +4519,24 @@ TR_CallSite* TR_InlinerBase::findAndUpdateCallSiteInGraph(TR_CallStack *callStac
 
    }
 
+#ifdef J9_PROJECT_SPECIFIC
+TR_PrexArgInfo* TR_PrexArgInfo::argInfoFromCaller(TR::Node* callNode, TR_PrexArgInfo* argInfo, TR_PrexArgInfo* callerArgInfo)
```

Comment:
I'm a bit confused about this API - why do we accept an argInfo and return an argInfo? Wouldn't it make more sense to allocate the TR_PrexArgInfo in here and just return it? This forces a somewhat strange convention of allocating in the caller to then assign the result of the propagation which I find confusing...